src/adaptors.h

```diff
@@ -2,7 +2,7 @@
 #define NODE_OR_TOOLS_ADAPTORS_F1FF74E9681C_H
 
 #include <nan.h>
-
+#include <iostream>
```

Comment:
remove unnecessary headers or declare a MACRO for debugging purposes. It should not be there on the release  builds.